modulemd/tests/test-modulemd-merger.c

```diff
@@ -12,6 +12,7 @@
  */
 
 #include <glib.h>
+#include <stdlib.h>
```

Comment:
I'm assuming you added this for `rand`. This can be eliminated by using the functionality in `glib`